src/main/java/org/folio/circulation/resources/RequestNoticeSender.java

```diff
@@ -237,7 +252,8 @@ private CompletableFuture<Result<Void>> sendNoticeForRequestWithoutItemId(Reques
     UUID templateId = templateIdExtractor.apply(tlrSettings);
 
     if (request.isTitleLevel() && tlrSettings.isTitleLevelRequestsFeatureEnabled() && templateId != null) {
-      return sendNotice(request, templateId, eventType);
+      return fetchLatestPatronInfoAddedComment(request)
```

Comment:
Looks like we are fetching patron info for every single patron notice we send using RequestNoticeSender. Can we try and centralize fetching patron info? For example, by introducing a new method that would wrap the call to `sendNotice()` and combine in with a call to a method that fetches additional data.